carmm/run/workflows/react.py

```diff
@@ -165,20 +162,22 @@ def aims_optimise(self,
                         unit_cell_relaxer = StrainFilter(self.initial)
 
                         opt = BFGS(unit_cell_relaxer,
-                                   trajectory=str(counter) + "_" + filename + "_" + str(opt_restarts) + ".traj",
+                                   trajectory="".join([subdirectory_name, "/", str(counter), "_",
+                                                       filename, "_", str(opt_restarts), ".traj"]),
```

Comment:
Perhaps some general purpose function for generating the directory path with a given extension would reduce redundancy (not a big issue tbqh).

Reply:
Aha, right you are! Found it further down.